app/controllers/api/avatars.php

```diff
@@ -349,7 +349,7 @@
             CURLOPT_USERAGENT => \sprintf(
                 APP_USERAGENT,
                 App::getEnv('_APP_VERSION', 'UNKNOWN'),
-                App::getEnv('_APP_SYSTEM_SECURITY_EMAIL_ADDRESS', APP_EMAIL_SECURITY)
+                App::getEnv('_APP_EMAIL_SECURITY', APP_EMAIL_SECURITY)
```

Comment:
The `APP_EMAIL_SECURITY` constant is an empty string so it's not really helpful 🧐 

For all of these changes, we should also fall back to `_APP_SYSTEM_SECURITY_EMAIL_ADDRESS` email address until 1.6.0.

```suggestion
                App::getEnv('_APP_EMAIL_SECURITY', App::getEnv('_APP_SYSTEM_SECURITY_EMAIL_ADDRESS', APP_EMAIL_SECURITY))
```